app/models/feedback.rb

```diff
@@ -0,0 +1,2 @@
+class Feedback < ApplicationRecord
```

Comment:
I wonder if there could be some security implications here...

I was wondering if this might be useful for other govt sites / projects, e.g. for transitions. Then I realised that it may be possible to arrive at the feedback endpoint from _any_ site (unless I'm misreading this PR). The fact that we have a link 'return to gov.au' suggests that we might in certain circumstances end up with people thinking they are returning to gov.au if they arrive from a different referrer ... should we have a validation whitelist / regex on the URL, perhaps?